macros/src/actix.rs

```diff
@@ -659,6 +659,44 @@ pub fn emit_v2_errors_overlay(attrs: TokenStream, input: TokenStream) -> TokenSt
     gen.into()
 }
 
+fn extract_rename(attrs: &[Attribute]) -> Option<String> {
+    attrs
+        .into_iter()
+        .filter(|attr| {
+            attr.path
+                .get_ident()
+                .map(|id| id.to_string() == "openapi")
+                .unwrap_or(false)
+        })
+        .next()
```

Comment:
This doesn't seem right, we should probably search the it as it might not always the first one? (we could add other things in the future).
Maybe have a look at `emit_v2_security` which uses the openapi attribute already.